server/app/onboarding.go

```diff
@@ -46,14 +46,14 @@ func (a *App) PrepareOnboardingTour(userID string, teamID string) (string, strin
 }
 
 func (a *App) getOnboardingBoardID() (string, error) {
-	boards, err := a.store.GetTemplateBoards(globalTeamID)
+	boards, err := a.store.GetTemplateBoards("0", "")
```

Comment:
Use globalTeamID instead of "0". Here and elsewhere.

Reply:
Done.